vms/pom/pages/basePage.py

```diff
@@ -14,14 +20,27 @@ class BasePage(object):
     def __init__(self, driver):
         self.driver = driver
 
+    # New function to check if Element is visible before calling it
+    def is_visible(self, element, timeout=5):
+        try:
+            wait(self.driver,
+                 timeout).until(ec.visibility_of_element_located((By.XPATH,
+                                                                  element)))
+            return True
+        except TimeoutException:
+            return NoSuchElementException
```

Comment:
Just return `False` your use of this function already raises this error on it's own

Reply:
The `TimeoutException` is raised when the element is not found even after 5s, if that happens then, it must be because Element is not there.
`TimeoutException` might be misleading when there's no such element instead.

Reply:
You are returning `NoSuchElementException` and not raising it, so returning False instead will also do the same thing as per your uses of this function in other places

Reply:
Okay sir got it. Sorry for this silly error. :sweat_smile:

Reply:
Only Monal is fine.